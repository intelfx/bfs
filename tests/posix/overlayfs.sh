test "$UNAME" = "Linux" || skip

cd "$TEST"
"$XTOUCH" -p lower/{foo,bar,baz} upper/{bar,baz/qux}

mkdir -p work merged
bfs_sudo mount -t overlay overlay -olowerdir=lower,upperdir=upper,workdir=work merged || skip
defer bfs_sudo rm -rf work
defer bfs_sudo umount merged

bfs_diff merged
