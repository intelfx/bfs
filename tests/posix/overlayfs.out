merged
merged/bar
merged/baz
merged/baz/qux
merged/foo
